src/Buttons/Button.jsx

```diff
@@ -3,18 +3,21 @@ import PropTypes from "prop-types";
 import ButtonStyles from "./button.style";
 
 export default function Button({
-  disabled,
-  onClick,
   type,
+  htmlType,
+  onClick,
   children,
+  disabled,
   customStyle
 }) {
   return (
     <ButtonStyles>
+      {/* eslint-disable-next-line */}
```

Comment:
https://github.com/yannickcr/eslint-plugin-react/issues/1555

Ignored the eslint error as it seems buggy imo. Would rather just have this one prop than three separate button components with the exact same content.

Reply:
when I remove this comment I dont get an eslint error. Mb double check if we need this

Reply:
So both of your comments you're not getting eslint errors - perhaps your linter doesn't pick up on some things? In the past on some of your PR's you missed some linted things so I'm skeptical of this.

Reply:
> So both of your comments ...

which is the other commend ?

Reply:
I mean I deleted one comment but that is irrelevant with the eslint. It was about the "\\e80f" thingy.

Reply:
> hmm weird, why did you delete that one though?

it makes sense that I add it again now that I am thinking of it. :)